src/main/java/nextstep/fp/Lambda.java

```diff
@@ -26,31 +30,21 @@ public void run() {
         }).start();
     }
 
+    private static int sumWithCondition(List<Integer> numbers, SumCondition condition) {
+        return numbers.stream()
+                .filter(n -> condition.test(n))
+                .reduce(0, (subtotal, n) -> subtotal + n);
```

Comment:
```suggestion
        return numbers.stream()
                .filter(condition::test)
                .reduce(0, Integer::sum);
```
다음과 같이 표현해볼 수도 있을 것 같아요! 😄